base/applications/rapps/lang/tr-TR.rc

```diff
@@ -5,6 +5,7 @@
  * TRANSLATORS: Copyright 2013-2016 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
  *              Copyright 2018 Ercan Ersoy (ercanersoy) <ercanersoy@ercanersoy.net>
  *              Copyright 2021 Süleyman Poyraz <zaryob.dev@gmail.com>
+ *              Copyright 2024 Ethem De Santa <ethemrty@gmail.com>
```

Comment:
Revert the part of commit https://github.com/reactos/reactos/pull/7465/commits/901377d6a6862ededec2808a785a2b1001e7abf0 where you change your real name to a nickname.